system/jlib/jflz.cpp

```diff
@@ -676,6 +682,9 @@ class jlib_decl CFastLZCompressor : public CInterface, public ICompressor
 
     virtual void open(void *buf,size32_t max)
     {
+        wrmax = max / 4;
+        if (wrmax < 256)
+            wrmax = max;
```

Comment:
I changed it so now I will try to write up to max bytes so no more divide by 4 with minimum